PennMobile/Login/LoginWebviewController.swift

```diff
@@ -70,7 +70,7 @@ class LoginWebviewController: PennLoginController, IndicatorEnabled {
     
     private func getDiningBalance() {
         if let student = Student.getStudent(), student.isFreshman() {
-            DiningViewModel.ShowDiningPlan = true
+            DiningViewModel.showDiningPlan = true
```

Comment:
I really really don't like this solution. It stems from us trying to ascertain whether a student is on a meal plan or not before we have actually tried to fetch their balance so that we can determine whether or not to show them the balance section on the dining page. Can we remove this line and change the dining page to show the balance section for all logged in users whether or not they have a balance?

Reply:
So show balance for everyone logged in? Would that be Student.getStudent() to determine if they are logged in or not?

Reply:
Adding a GitHub issue, this will be in a different PR